### ISSUE TITLE
Models get worse after finetuning

### ISSUE THREAD

**Issue by HenryL27** [UKPLab/sentence-transformers#1891]

**Comment by HenryL27** [comment]
Oh, here's the GPL repo I'm referring to: [https://github.com/UKPLab/gpl](gpl)

**Comment by HenryL27** [comment]
I also wrote a MarginMSE Evaluator, and handed it a subset of the training data. Interestingly, it appears that the loss function values start increasing (!) after a couple hundred steps. So something is very very wrong, right?

**Comment by HenryL27** [comment]
nope. you?
